2001-3000/LIT2393tabiba.xml

```diff
@@ -56,6 +56,14 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <citedRange unit="page">118-163</citedRange>
                </bibl>
             </listBibl>
+            <div type="bibliography">
```

Comment:
there is something wrong with the nesting, you already had the `<div type="bibliography">` opened in line 48 above, please remove line 59